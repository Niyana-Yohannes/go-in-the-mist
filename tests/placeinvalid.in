place A1
who
place G1
who
place A1  
place    A1
place A11111111111111
place A11111111111111 
place A11111111111111 A1111111
place 
place A
place 1
place A20
place 20A
place lol
place 200
