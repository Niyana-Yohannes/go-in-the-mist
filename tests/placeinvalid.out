W
B
Invalid!
Invalid!
Invalid coordinate
Invalid!
Invalid!
Invalid!
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
